net/baresip/Config.in

```diff
@@ -0,0 +1,16 @@
+menu "bareSIP configuration"
+	depends on PACKAGE_baresip
+
+config BARESIP_WITH_FFMPEG
+	bool "Compile with FFmpeg support"
+	default y if (i386||x86_64)
```

Comment:
I think this should be `default y if TARGET_x86` or `default TARGET_x86` instead. Iirc there is no `CONFIG_x86_64` symbol in LEDE / OpenWrt.